filename_in: "/box_005/006/box_db_10__0_filtered_0.csv"
filename_model: "/box_005/006/model_40/model_40"
filename_out: "/box_005/006/"